### PR TITLE
[Feat] Add litellm.ContentPolicyViolationError 

### PR DESCRIPTION
**Pull request by ishaan-jaff**
Easily catch these errors for dalle-3, azure

### CODE REVIEW

**Comment by ishaan-jaff** [comment]
addressing: https://github.com/BerriAI/litellm/issues/1372

File: litellm/main.py [hunk 1]
```diff
@@ -1117,7 +1117,7 @@ def completion(
                 acompletion=acompletion,
                 logging_obj=logging,
                 custom_prompt_dict=custom_prompt_dict,
-                timeout=timeout
+                timeout=timeout,
             )
```
Comment (ishaan-jaff) [inline litellm/main.py:1121]: black8 formatting

**Comment by ishaan-jaff** [comment]
merging - this passes ci/cd testing !

**Comment by ghost** [comment]
❤️ ❤️

**Comment by ghost** [comment]
@ishaan-jaff can you document this here - https://docs.litellm.ai/docs/exception_mapping

So people know it's available

**Pull request merged**